### ISSUE TITLE
add support for attribute {.scala} to verbatim text

### ISSUE THREAD

**Issue by michelou** [scala/scala3#11014]
@abgruszecki [Scala3doc](https://github.com/lampepfl/scala3doc) currently supports the following Markdown syntax for code blocks :
````
```scala
val x = 1
```
````

Can you please validate that [Scala3doc](https://github.com/lampepfl/scala3doc) does also support the following two Markdown syntaxes (see discussion in PR [#10953](https://github.com/lampepfl/dotty/pull/10953))  ?

1. Code block with attribute `{.scala}`, for instance :
````
```{.scala}
val x = 1
```
````
2. Verbatim text with attribute `{.scala}`, for instance :
````
Given the dedinition `val x = 1`{.scala}
````

**Goal**: display the same Scala syntax hightlighting both in code blocks and in verbatim texts.

**Comment by abgruszecki** [comment]
I agree it'd be useful for inline monospace and code blocks to have the same support for highlighting. We probably should take a look at some extensions to Markdown that support that sort of syntax.

/cc @romanowski
